### PR TITLE
Adapting to the new name

### PR DESCRIPTION
**Pull request by veni-vidi-code**
## Summary

It just changes the name and all the links

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by redcreeper14385 — APPROVED** [review]
Looks good 👍

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ The following is a set of guidelines for contributing to the repository. These a
 
 Generally speaking questions are better suited in our resources below.
 
-- The official support server: https://discord.gg/r3sSKJJ
+- The official support server: https://discord.gg/ZebatWssCB
 - The Discord API server under #python_discord-py: https://discord.gg/discord-api
-- [The FAQ in the documentation](https://discordpy.readthedocs.io/en/latest/faq.html)
-- [StackOverflow's `discord.py` tag](https://stackoverflow.com/questions/tagged/discord.py)
+- [The FAQ in the documentation](https://nextcord.readthedocs.io/en/latest/faq.html)
+- [StackOverflow's `nextcord` tag](https://stackoverflow.com/questions/tagged/nextcord)
```
Comment (TAG-Epic) [inline .github/CONTRIBUTING.md:14]: I think we should leave the discord.py tag alone

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[build-system]
```
Comment (TAG-Epic) [inline pyproject.toml:1]: This file should not be here?

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
Else it looks pretty good to me

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (veni-vidi-code) [reply .github/CONTRIBUTING.md:14]: I was thinking about that. Maybe mention both?

File: pyproject.toml (hunk 2 above)
Comment (veni-vidi-code) [reply pyproject.toml:1]: Oh, forgot to add it to gitignore

Comment (veni-vidi-code) [reply pyproject.toml]: Should be fixed

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (TAG-Epic) [reply .github/CONTRIBUTING.md:14]: Good idea.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -8,9 +8,10 @@ The following is a set of guidelines for contributing to the repository. These a
 
 Generally speaking questions are better suited in our resources below.
 
-- The official support server: https://discord.gg/r3sSKJJ
+- The official support server: https://discord.gg/ZebatWssCB
 - The Discord API server under #python_discord-py: https://discord.gg/discord-api
-- [The FAQ in the documentation](https://discordpy.readthedocs.io/en/latest/faq.html)
+- [The FAQ in the documentation](https://nextcord.readthedocs.io/en/latest/faq.html)
+- [StackOverflow's `nextcord` tag](https://stackoverflow.com/questions/tagged/nextcord)
 - [StackOverflow's `discord.py` tag](https://stackoverflow.com/questions/tagged/discord.py)
```
Comment (veni-vidi-code) [inline .github/CONTRIBUTING.md:15]: @TAG-Epic

**Review by TAG-Epic — APPROVED** [review]
LGTM

**Pull request merged**